e2e/tests/config/mod.rs

```diff
@@ -0,0 +1,5 @@
+pub(crate) const ALPINE_WITH_EXTENSION_0_5_4_PREFIX: &str =
+    "timescaledev/promscale-extension:0.5.4-ts2.7.0-pg";
+pub(crate) const HA_WITH_EXTENSION_0_5_4_PG14: &str = "timescale/timescaledb-ha:pg14.4-ts2.7.1-p2";
```

Comment:
@sumerman I'm not entirely happy with having the version number in the name of the constants here. On the one hand I want it to be obvious what the referenced version is, but on the other hand it means that the constant needs to be renamed every time the image is changed, which might not actually happen. WDYT? Any other ideas?

Reply:
I'm inclined to name them `*_WITH_EXTENSION_LAST_RELEASED_*`, but I'm also fine with leaving them as they are and renaming them if we find ourselves too annoyed by it.

Reply:
done